.github/workflows/requirements.txt

```diff
@@ -1,6 +1,5 @@
-meson==1.6.1 \
-    --hash=sha256:1eca49eb6c26d58bbee67fd3337d8ef557c0804e30a6d16bfdf269db997464de \
-    --hash=sha256:3f41f6b03df56bb76836cc33c94e1a404c3584d48b3259540794a60a21fad1f9
+meson==1.7.2 \
+    --hash=sha256:82c6818dc81743c96de3a458f06175776ebfde4081195ea31ea6971838f25e38
```

Comment:
The meson failures were spotted in https://github.com/evverx/systemd/pull/245. It was fixed in https://github.com/mesonbuild/meson/commit/7889cac7d5c976a8f11c47bfcb8978ac27b15a61.